src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int MAX_FIGURE_LENGTH = 7;
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[MAX_FIGURE_LENGTH];
+
+        for (int i = 0; i < figures.length; i++) {
+            if (i < figures.length / 2) {
+                figures[i] = figureSupplier.getRandomFigure();
+                figures[i].draw();
+            } else {
+                figures[i] = figureSupplier.getDefaultFigure();
+                figures[i].draw();
```

Comment:
Don't return `null` from a method. Ensure that `getRandomFigure()` and `getDefaultFigure()` in `FigureSupplier` class do not return `null`. If they are implemented using a `switch case construction`, the last possible option should be in the `default` case.